webview-ui/src/utils/vscStyles.ts

```diff
@@ -0,0 +1,39 @@
+export const INPUT_BACKGROUND_VAR = "--vscode-input-background"
+export const BACKGROUND_VAR = "--vscode-sideBar-background"
+export const FOREGROUND_VAR = "--vscode-editor-foreground"
+export const FOREGROUND_MUTED_VAR = "--vscode-foreground-muted"
+export const DESCRIPTION_FOREGROUND = "--vscode-descriptionForeground"
```

Comment:
Hey Rob I don't like how we suffix these variables with 'VAR' although i know continue does this. Lets also prefix with VSC